network/src/connectivity_manager/test.rs

```diff
@@ -233,8 +235,10 @@ fn connect_to_seeds_on_startup() {
         conn_mgr_reqs_tx
             .send(ConnectivityRequest::UpdateAddresses(
                 DiscoverySource::Gossip,
-                seed_peer_id,
-                vec![new_seed_addr.clone()],
+                [(seed_peer_id, vec![new_seed_addr.clone()])]
+                    .iter()
+                    .cloned()
```

Comment:
does this work with `.into_iter()` instead of `.iter().cloned()`?

Reply:
> value of type `std::collections::HashMap<move_core_types::account_address::AccountAddress, std::vec::Vec<libra_network_address::NetworkAddress>>` cannot be built from `std::iter::Iterator<Item=&(move_core_types::account_address::AccountAddress, std::vec::Vec<libra_network_address::NetworkAddress>)>`